.github/workflows/image-build-pdf-service.yaml

```
@@ -36,7 +36,7 @@
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
           npx sonar-scanner -Dsonar.projectKey=solutions_pdf_service -Dsonar.organization=solutions -Dsonar.host.url=https://sonarcloud.io -Dsonar.token=${{ secrets.SONAR_TOKEN }} -Dsonar.sources=src -Dsonar.exclusions="node_modules/**,dist/**"
-        working-directory: common/pdf-service
+        orking-directory: common/pdf-service
```

actionlint (1.7.4)

unexpected key "orking-directory" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)